common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3546,4 +3576,13 @@ object CelebornConf extends Logging {
       .booleanConf
       .createWithDefault(true)
 
+  val ACTIVE_STORAGE_LEVELS: ConfigEntry[String] =
+    buildConf("celeborn.storage.activeLevels")
+      .categories("master", "worker")
+      .version("0.3.0")
+      .doc("Enabled storage levels. Optional values: HDD,SSD,HDFS. ")
```

Comment:
question: how do you treat S3, OSS? independent options, or alias of HDFS?

Reply:
nit: "Optional values" => "available options", keep consistent with others

Reply:
S3, OSS and other storage levels will be added when ready.